.buildkite/hooks/pre-command

```diff
@@ -72,3 +72,7 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-agent-binary-dra" ]]; then
     release_manager_login
   fi
 fi
+
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-agent" && "$BUILDKITE_STEP_KEY" == "integration-fips-cloud-image" ]]; then
```

Comment:
For the future me, I think we can do something about a BK plugin, so we can be explicit about the pipeline when the  docker login is needed, so we avoid using `pre-command` hooks with the combination of `BUILDKITE_STEP_KEY` and `BUILDKITE_PIPELINE_SLUG` for the sake of simplicity